packages/bpk-component-badge/src/BpkBadge.js

```diff
@@ -45,12 +63,21 @@ const BpkBadge = props => {
 };
 
 BpkBadge.propTypes = {
+  type: PropTypes.oneOf([
+    BADGE_TYPES.WARNING,
+    BADGE_TYPES.SUCCESS,
+    BADGE_TYPES.DESTRUCTIVE,
+    BADGE_TYPES.LIGHT,
+    BADGE_TYPES.INVERSE,
+    BADGE_TYPES.OUTLINE,
+  ]),
```

Comment:
As discussed above, if `BADGE_TYPES` matches keys and values, this can become `oneOf(Object.keys(BADGE_TYPES))`.